examples/Notebook.ipynb

```diff
@@ -133,7 +133,7 @@
    "source": [
     "from jupytercad import CadDocument\n",
     "\n",
-    "doc = CadDocument(\"examples/example3.FCStd\")\n",
```

Comment:
This path depends on where you start `jupyterlab`

Reply:
It used to but it should have been fixed in JupyterLab 4.0.1 https://github.com/jupyterlab/jupyterlab/commit/0d7a24c86dcd328d3d2a59ad2e736c5ea71a81b3

Reply:
ah ok, nice!!!